lib-wrapper/android-bindings/publish.gradle

```diff
@@ -1,6 +1,6 @@
 apply plugin: 'maven-publish'
 
-version '6.0.0-pre1'
+version '6.0.0'
```

Comment:
Just so I'm sure this is 6.0.0 for the clients?
Will there be another android version when the MobileCoinFoundation releases 6.0.0, with the enclave values?

Reply:
This version is just for the SDK bindings. It loosely follows the Mobilecoin core version, but doesn't match exactly. I plan on tagging 6.0 releases for Android after merging this to get the release cycles started for updating Android apps. New versions will be released for core updates as needed.